src/NotificationSystem.jsx

```diff
@@ -162,6 +162,26 @@ var NotificationSystem = React.createClass({
     });
   },
 
+  deleteNotification: function(notification) {
+    var self = this;
+    var uid = notification.uid ? notification.uid : notification;
+    Object.keys(this.refs).forEach(function(container) {
+      if (container.indexOf('container') > -1) {
+        Object.keys(self.refs[container].refs).forEach(function(_notification) {
+          if (_notification === 'notification-' + uid) {
+            console.log('remove ', _notification);
+            self.refs[container].refs[_notification]._removeNotification();
+          }
+        });
+      }
+    });
```

Comment:
Can you reuse `getNotificationRef` from https://github.com/igorprado/react-notification-system/pull/89/files ?